### ISSUE TITLE
Enrichment vignette: suggest we remove the p-value bar chart and pie chart

### ISSUE THREAD

**Issue by yochannah** [intermine/InterMineR#62]
https://bioconductor.org/packages/release/bioc/vignettes/InterMineR/inst/doc/Enrichment_Analysis_and_Visualization.html#piechart-and-barplot-graphs

Pies and bar charts are pretty, but given the fact that a smaller P value is better, it takes some figuring out to understand what's going on - the smaller the pie segment / the smaller the bar, the more significant it is. 

One option could be to subtract all the values from 1 so smaller values get bigger slices, but that would also require some explaining. I think it might make more sense just to remove these two visualisations.

**Comment by rachellyne** [comment]
Yep, you don't have to use the p-value.  I think you can set the orderby to other things like geneNum.  I think that's what we do in the Python tutorial for similar graphs and what we did last time.

**Comment by splaisan** [comment]
or use -log(pval) like in many other plots out there! :-)

**Comment by yochannah** [comment]
@splaisan that would make a lot more sense, too!

**Comment by yochannah** [comment]
I've added the hacktoberfest label to this - interested contributors, if you're interested in helping out, the chart in question is here: https://github.com/intermine/InterMineR/blob/master/vignettes/Enrichment_Analysis_and_Visualization.Rmd#L300 

## if you want to pick up this task

1. Comment on this issue stating that you intend to work on the task
2. When you're ready, add your work to the repo and create a pull request.

### What to do if you need help

Mention @yochannah, tweet [@yoyehudi](http://www.twitter.com/yoyehudi), pop by to say hi on [chat](http://chat.intermine.org) or if needed email yo@intermine.org <3 note that we're only usually in the office during UK office hours and may not answer outside these times.
